scaleway/helpers_object.go

```diff
@@ -92,3 +94,7 @@ func expandObjectBucketTags(tags interface{}) []*s3.Tag {
 
 	return tagsSet
 }
+
+func endpointURL(bucketName string, region scw.Region) string {
```

Comment:
Maybe rename it `objectEndpointURL` as it is shared in the whole `scaleway` package?